### DISCUSSION TITLE
Splitting Seasonal Races Up

### DISCUSSION THREAD

**Discussion by Blasta-alasta** [sunflower-land/sunflower-land#2837] [Ideas]
**_NOTE: I’m not advocating for changing the season, just applying an idea and using this season’s items and gameplay as an example of how splitting the races could work._**

_**In my opinion a few issues arise from very low supply items. I believe below ideas could work, but would work better if the supply was higher for some items. 100 Supply with 15,000+ players is still extremely exclusive. Ignoring boosts in my opinion the supply for the Infernal Pitchfork should have been more like 100, Luna’s Hat 200, Cattlegrim 400, Queen is fine at 1000.**_

**### THIS SEASONS TOP ITEMS**

**Capped Supply Boost Items:**
Infernal Pitchfork		20 (supply)					
Luna’s Hat			50				
Cattlegrim			200				
Queen Cornelia		1000				

**Capped Supply Cosmetic Items:**										
Whale Hat			20				
Crumple Crown		260
Infernal Horns			400
Infected Potion		1200

### ALTERNATIVE APPROACH

Gain Feathers from doing chores (uncapped), and deliveries (capped). Remove feathers from Auctions (or only use high value cosmetics for feather auctions). Split supply of high value boost items in three allocations:
(Allocation 1 / 2 / 3)

Infernal Pitchfork: 10 / 10 / 0		
Luna’s Hat: 20 / 20 / 10			
Cattlegrim:  75 / 75 / 50		
Queen Cornelia: 400 / 400 / 200

**Allocation 1: Auctions**

SFL (or resource auctions) - straight forward, no explanation needed. Appeals to our whales who want to battle it out with their wallets.

**Allocation 2: Grinding Race**
			       Supply	         Feathers	Other Cost		
Infernal Pitchfork:
Supply allocation: 10
Cost: 4000 Feathers / 100 Gold / 2000 Corn		

Luna’s Hat:
Supply allocation: 20
Cost: 4000 Feathers  / 50 Gold / 1500 Corn

Cattlegrim:		
Supply allocation: 75
Cost: 3000 Feathers / 75 Gold / 1000 Corn

Queen Cornelia:
Supply allocation: 400
Cost: 2000 Feathers / 50 Gold / 500 Corn

_(You could include an allocation for Whales Hat, Crumple Crown, Infernal Horns, & Infected potion if they were removed from the non SFL auctions.)_

This appeals to grinders & hoarders and forces them to burn, and manage both their resources and chores. A challenge being harvesting enough corn as well as completing the chores as fast as possible.

**Allocation 3: RNG**

Have a separate system/game based on RNG that all players (banner holders) can partake in… this could potential drop a second type of ticket, or even something similar to the Golden Crop competition.

### IDEAS FOR THIS:

**Loyalist NPCs**
Example: Every Player that complete all of Pete Chores but none of Tywins goes into the draw too win a prize. This could be done with the SFL giving NPCs too.

You could make them exclusive, like if you only visited Bert & Pete everyday, and not Tywin or Cornwall you could go into Bert & Pete’s special draw

Or

If you complete them all. If you complete all of Pete’s deliveries you going Petes special draw. If you do all of Pete and Ravens, then your in Pete and also Raven’s draw. The prizes could scale with the NPC’s difficulty.

**Golden Crop**
What if during the season all players could have a chance to find a rare golden crop. The golden crop could be exchange at a new NPC house to enter one of a series of raffles. So if a player finds golden crop they can choose which raffle to enter. Multiple golden crops could be used for multiple entries.

**A Second Ticket**
Have a second ticket that only drops from planting crops as the feathers do now. Increase the average a bit to maybe 4-6 tickets a day. These ticket can then be used to mint the items in the RNG allocation, or you could even use them to auction off the items in this allocation, instead of the ones coming from completing deliveries and chores.

**Do it with Treasure Island**
Introduce another TI item(s) that all can dig for. This/these can be used to mint items in this allocation. (The only issue I see this is the unfair advantage HoDJ gives, but I’m sure its possible to cap the drops at the first 30 digs?

**ALTERNATIVE IDEA:**

Scape the allocation and instead do a prizes for new players from the treasury. You could set a benchmark and a limit, like every player that runs through their lists of chores once (but only once) qualifies for a special raffle with more beginner appraise items and/or resources from the Treasury.

### OTHER SEASONAL ITEMS
 
These time limited items are perfect and similar could be used in the same format. They provide back up options for players that not only lose out in the grind race, but also those don’t like grinding as hard, or those low to mid level player trying to get a taste of the competitive part of the season.

Poppy 250 (Feathers)
Kernaldo	500
Grain Grinder	750

Infernal B Potion	n/a
Internal G Potion	n/a
Witching Wardrobe	 50
Witches Broom 50
Ox Suit 500
Imp Costume 1000
Crow Wings 3000

Candles 10
Haunted Stump 100
Spooky Tree 500

**Comment by TheDcol** [comment]
Huge write up Blasta 💙

Can i ask what you mean by 'uncapped chores'? Do you mean like it was in prior seasons?

**Comment by alisa-yar** [comment]
I think this suggestion/idea can solve the majority of the issues that players have with the seasonal tickets systems (both the regular RNG one and the new one that’s based on the factions leaderboard.

**Reply by Blasta-alasta** [reply]
Gosh this was an old post Alisa!

I think the dream that any old player can grind hard enough and win a top item is pretty gone now. And same with uncapped chores, too many people complained about those.

It's pretty clear that the team wants to set the standard that the top auction items are for the most advanced players with the very top ones being pretty much strictly pay to win. But, there is a supply of mid-tier lemon and tomato ones (in this season for example)  that I think mid level farmers could aspire to. And with more options in Stella's much has improved in regards to options for those that cant compete or just miss out.

I hear that Stella's items and also the Pharoah's seasonal items may be added to the Treasure/Rare/Luxery Chests too next month. So auctions aside, there will be a RNG chance for any player to win these which is quite cool and going along the lines of some of what was suggested above and elsewhere.

And we have the new digging with the time limited collectables which most players can go for, even it also kinda pay to win, at least you profit back with coins, especially if saving for the Camel.
